src/averageAges.js

```diff
@@ -20,6 +20,20 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const manArr = century
+    ? people.filter(
+      person =>
+        (person.sex === 'm') && (Math.ceil(person.died / 100) === century)
+    )
+    : people.filter(
+      person =>
+        person.sex === 'm'
+    );
```

Comment:
```suggestion
  const men = people.filter(person =>
    century
      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
      : person.sex === 'm'
  );
```

The same but a bit shorter :)